pages/css/components/toasts.md

```diff
@@ -88,8 +88,8 @@ To create a default toast, use `.Toast`
 
 ```html title="Toast animating"
 <div class="p-3">
-  <div class="Toast Toast-animated Toast-block-error">
-    <span class="Toast-octicon">
+  <div class="Toast Toast--error Toast--animateIn">
```

Comment:
Should the animation be a separate class? Or should they all automatically animate?

Reply:
@simurai and I chatted last week about the animation being a modifier, I think it makes more sense to leave it up to the user in that regard (how/when to add the class). Unsure about the classname, defer to what you think is best @shawnbot @simurai

Reply:
> Should the animation be a separate class?

Maybe we can revisit this after we see 2-3 use cases. It also depends a bit how a `Toast` gets added to the page. On page load and initially hidden or injected only when needed?

> @shawnbot I think the classname should be Toast--animate-in, no?

Our [naming convention](https://primer.style/css/principles#bem-style-naming-and-structure) doesn't specify the format when using **multiple words** for a modifier. So my thinking was to use camelCase to not make it look like the modifier has a child element. SUIT CSS uses the same [convention](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentname--modifiername).